scripts/checkout-khronos-gltf-samples.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+##
+# Copyright 2019 Google Inc. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+##
+
+# This script is responsible for cloning and / or updating the local repository
+# of Khronos glTF sample models
+
+REPO_URL=https://github.com/KhronosGroup/glTF-Sample-Models.git
+CLONE_PATH=./examples/assets/glTF-Sample-Models
+
+if [ ! -d "$CLONE_PATH/.git" ]; then
```

Comment:
Maybe a warning here, or in the README, that this will pull down a large amount of data

Reply:
Good call, will fix 👍

Reply:
There is a warning in both places now.